docs/examples/fork-process-model/flask-gunicorn/gunicorn.config.py

```diff
@@ -0,0 +1,29 @@
+from opentelemetry import trace
+from opentelemetry.exporter.jaeger import JaegerSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchExportSpanProcessor
+
+bind = "127.0.0.1:8000"
```

Comment:
```suggestion
BIND = "127.0.0.1:8000"
```

Same for the module-level constants below.

Reply:
These should be lowercase otherwise Gunicorn doesn't pick them up.